libs/blocks/quiz/utils.js

```diff
@@ -497,3 +498,17 @@ export const getAnalyticsDataForLocalStorage = (config) => {
 export const isValidUrl = (url) => VALID_URL_RE.test(url);
 
 export const getNormalizedMetadata = (el) => normalizeKeys(getMetadata(el));
+
+export const getLocalizedURL = (originalURL) => {
+  const { locale } = getConfig();
+  const prefix = locale?.prefix;
+  const localCode = locale?.ietf ?? 'en-US';
+
+  let localizedURL = originalURL;
+
+  if (localCode !== 'en-US' && !originalURL.startsWith(`${prefix}/`)) {
+    localizedURL = `${prefix}${originalURL}`;
+  }
+
+  return localizedURL;
```

Comment:
I think we can do a more concise version for this:
```suggestion
  const { locale } = getConfig();
  const prefix = locale?.prefix;

  return locale?.ietf !== 'en-US' && !originalURL.startsWith(`${prefix}/`)
  ? `${prefix}${originalURL}`
  : originalURL;
```

Reply:
If locale.ietf not define, your code cannot handle it

Reply:
I see. How about this, using destructuring and being even more concise:

```suggestion
  const { locale } = getConfig();
  const { prefix, ietf = 'en-US' } = locale || {};
  return ietf !== 'en-US' && !originalURL.startsWith(`${prefix}/`) ? `${prefix}${originalURL}` : originalURL;
```